regression/cbmc/va_list3/windows-preprocessed.desc

```diff
@@ -0,0 +1,8 @@
+CORE broken-smt-backend
+windows-preprocessed.i
+--winx64
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+--
+^warning: ignoring
```

Comment:
Can you add a comment, say in the .desc file, how windows-preprocessed.i was produced?

Reply:
It was kind-of noted in the commit message, but I'll add a note in the .desc file as well!